pkg/graph/resolvers/trb_request.go

```diff
@@ -222,6 +222,87 @@ func UpdateTRBRequestTRBLead(
 	return updatedTrb, err
 }
 
+// CloseTRBRequest closes a TRB request and sends an email if recipients are specified
+func CloseTRBRequest(
+	ctx context.Context,
+	store *storage.Store,
+	emailClient *email.Client,
+	fetchUserInfo func(context.Context, string) (*models.UserInfo, error),
+	fetchUserInfos func(context.Context, []string) ([]*models.UserInfo, error),
+	id uuid.UUID,
+	reasonClosed string,
+	notifyEUAIDs []string,
+) (*models.TRBRequest, error) {
+	// Query the TRB request, attendees in parallel
+	errGroup := new(errgroup.Group)
+
+	// Query the TRB request
+	var trb *models.TRBRequest
+	var errTRB error
+	errGroup.Go(func() error {
+		trb, errTRB = store.GetTRBRequestByID(ctx, id)
+		return errTRB
+	})
+
+	if errG := errGroup.Wait(); errG != nil {
+		return nil, errG
+	}
```

Comment:
Is it necessary to make this error group? Seems like we make it an immediately wait on the response